examples/groups/group.yaml

```diff
@@ -2,11 +2,12 @@ apiVersion: groups.gitlab.crossplane.io/v1alpha1
 kind: Group
 metadata:
   name: example-group
+  annotations:
+    crossplane.io/external-name: "GROUP ID"

```

Comment:
Is there a specific reason the `annotation` was added to the example? If it's not strictly needed, I believe we should omit it

Reply:
I think the annotation is necessary, since there is no way to create top-level groups, we can only load a group from gitlab by id

Reply:
Removed annotations